libs/designsystem/modal/src/modal/services/alert.helper.spec.ts

```diff
@@ -86,5 +87,14 @@ describe('AlertHelper', () => {
       expect(backdrop).toHaveComputedStyle({ '--backdrop-opacity': backdropOpacity });
       await ionModalElement.dismiss();
     });
+
+    it('should set alert title text content as modal label', async () => {
+      await TestHelper.whenReady(ionModal);
+      const dialogElement = ionModal.shadowRoot.querySelector('[role="dialog"]');
+
+      await TestHelper.whenTrue(() => dialogElement.hasAttribute('aria-label'));
+
+      expect(dialogElement.getAttribute('aria-label')).toBe(title);
```

Comment:
Consider using built in matcher:
```suggestion
      expect(dialogElement).toHaveAttribute('aria-label', title);
```

Reply:
toHabeAttribute does not exist here in the context of a spectator service factory, but will change it in the other places! 🙏